lib/identity_cache.rb

```diff
@@ -56,15 +66,9 @@ class << self
     self.fetch_read_only_records = version >= Gem::Version.new("0.5")
 
     def included(base) #:nodoc:
```

Comment:
In this case I don't want to use the included block since I still want to prevent the module from being included into a base and derived class.  ActiveSupport::Concern in this module is being used to manage the dependant modules. I've added `super` to this included method override though.